clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClient.groovy

```diff
@@ -198,15 +203,27 @@ class DockerRegistryClient {
    * This method will hit the /v2/ endpoint of the configured docker registry. If it this endpoint is up,
    * it will return silently. Otherwise, an exception is thrown detailing why the endpoint isn't available.
    */
+
   public void checkV2Availability() {
+    try {
+      doCheckV2Availability()
+    } catch (RetrofitError error) {
+      LOG.warn "checkV2Availability", error
```

Comment:
This will lead to a lot of wasted log space; we shouldn't `warn` everytime the registry requires authentication (I expect most registries will).